src/org/traccar/database/AttributesManager.java

```diff
@@ -26,9 +26,14 @@ public AttributesManager(DataManager dataManager) {
     }
 
     @Override
+    public Attribute getById(long calendarId) {
```

Comment:
I'm thinking now... can't we just change  `ExtendedObjectManager` to be generic?